bench.py

```diff
@@ -37,6 +70,42 @@ def calculator(test, arg, n_runs, only_result, workpath=os.getcwd()):
             statistics.median(w),
             statistics.pstdev(w)),
         bold=True)
+
+    if web is True:
+        commit = get_latest_commit('Parth-Vader', 'scrapy-bench')
```

Comment:
Shall it be commit of the benchmark repo (this), or commit of the scrapy repo (which is being benchmarked)?